docs/en/7-runtime.md

```diff
@@ -89,6 +89,31 @@ block('button')({
 });
 ```
 
+You can use custom mode (with name `size` for example) and its call `apply('size')` for redifininig on different levels or different conditions (with `match`).
+
+Suppose you have different levels: common, touch and desktop. On common level you write:
+
+```js
+block('my-block')({
+    size: 'm',
+    content: () => ({
+        block: 'button',
+        size: apply('size')
+    })
+});
+
+On touch level:
+
+```js
+block('my-block').mode('size')('l');
+```
+
+And on desktop level:
+
+```js
+block('my-block').mode('size')('s');
+```
+
 *Result of templating:*
```

Comment:
Move this result upper for previous example and templating result for this example here.